verso/src/main.rs

```diff
@@ -0,0 +1,16 @@
+use std::{env::current_exe, thread::sleep, time::Duration};
+
+fn main() {
+    let versoview_path = current_exe().unwrap().parent().unwrap().join("versoview");
+    let controller = verso::VersoviewController::new(
+        versoview_path,
+        url::Url::parse("https://example.com").unwrap(),
+    );
+    sleep(Duration::from_secs(10));
+    dbg!(controller
+        .navigate(url::Url::parse("https://docs.rs").unwrap())
+        .unwrap());
+    loop {
+        sleep(Duration::MAX);
```

Comment:
Do you plan to add something else in the future? For example, a receiver of IPC channel that could receive message from the process it creates.

Reply:
Yes, I would like to add some event listeners, and maybe some other things like network request handlers